core/shared/src/main/scala/org/http4s/package.scala

```diff
@@ -17,15 +17,19 @@
 package org
 
 import cats.data._
+import com.comcast.ip4s
 import fs2.Stream
 import org.typelevel.ci.CIString
+import org.typelevel.scalaccompat.annotation._
 
 package object http4s {
 
   type AuthScheme = CIString
 
   type EntityBody[+F[_]] = Stream[F, Byte]
 
+  val EmptyBody: EntityBody[Nothing] = Stream.empty[Nothing]
```

Comment:
\+

Reply:
should be removed, since we have the new entity model here